docs/tutorials/stateless-application/hello-minikube.md

```diff
@@ -116,6 +116,12 @@ Verify that `kubectl` is configured to communicate with your cluster:
 kubectl cluster-info
 ```
 
+Open the kubernetes dashboard GUI in browser:
+
```

Comment:
Capitalize Kubernetes.
in browser -> in a browser
Suggestion: Remove GUI.

Open the Kubernetes dashboard in a browser:

Reply:
@steveperry-53 Thanks for the comment. I've addressed it. Please see latest commit.